docs/lib/posixlib.rst

```diff
@@ -202,6 +202,7 @@ C Header          Scala Native Module
 .. _scala.scalanative.posix.sys.time: https://github.com/scala-native/scala-native/blob/master/posixlib/src/main/scala/scala/scalanative/posix/sys/time.scala
 .. _scala.scalanative.posix.sys.types: https://github.com/scala-native/scala-native/blob/master/posixlib/src/main/scala/scala/scalanative/posix/sys/types.scala
 .. _scala.scalanative.posix.sys.uio: https://github.com/scala-native/scala-native/blob/master/posixlib/src/main/scala/scala/scalanative/posix/sys/uio.scala
+.. _scala.scalanative.posix.sys.wait: https://github.com/scala-native/scala-native/blob/master/posixlib/src/main/scala/scala/scalanative/posix/sys/wait.scala
```

Comment:
@LeeTibbert this is the extra line I need right?

Reply:
I think all is good here.  I pulled javalib.rst. sphinx-build runs without an error related to
that file. When I use a browser (firefox) to view the generated html page, the link
is active. Clicking on it causes it to be followed.   Because it is a link to new
code, the target of the link does not yet exist, but that is an "expected" error until
this PR is merged.